### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by Phjmroberts** [skyline-emu/skyline#1618] [Ideas]
Hello I was wondering I don't have lot of normal money but I would like to still help out skyline is there a way I can donate my Google money I would  donate that all the time and am sure lots of others will to

**Comment by dima-xd** [comment]
Not now, but in a future you will be able to donate

**Reply by chostdz** [reply]
Am willing to donate too

**Comment by Phjmroberts** [comment]
Thank you you all are doing a great job you are putting out exciting stuff
